### PR TITLE
Cusdk 54 pypi deployment

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
PyPI deployment has been added to Travis, and Packager and deployment scripts have been cleaned up.

There's also a small bugfix with the return type of Dictionary.fromObject_r method.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,13 +30,5 @@ deploy:
     branch: master
 
 after_deploy:
-  - export GPG_TTY=$(tty)
```
Comment (JaviCerveraIngram) [inline .travis.yml]: This has been moved to stuff/maven_deploy.sh.

File: Packager.hx [hunk 2]
```diff
@@ -245,16 +230,23 @@ class Packager {
...
-        var file = sys.io.File.write('_packages/connect.py/setup.py');
+        final file = sys.io.File.write('$outDir/setup.py');
+
+        file.writeString("from os import path" + EOL);
         file.writeString("from setuptools import setup" + EOL + EOL + EOL);
+        file.writeString("with open(path.join(path.abspath(path.dirname(__file__)), 'README.md')) as fhandle:" + EOL);
+        file.writeString("    README = fhandle.read()" + EOL + EOL + EOL);
         file.writeString("setup(" + EOL);
-        file.writeString("    name='connect'," + EOL);
+        file.writeString("    name='connect-sdk-haxe-port'," + EOL);
```
Comment (JaviCerveraIngram) [inline Packager.hx]: This is the package name suggested by Max & Marc.

File: stuff/PYTHON_README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# CloudBlue Connect Python SDK
```
Comment (JaviCerveraIngram) [inline stuff/PYTHON_README.md]: This file is set as the package description on PyPI.

File: stuff/pypi_deploy.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+sudo apt install python3-pip -y
+sudo -H pip3 install --upgrade pip
+echo "*** Installing wheel..."
+sudo -H pip3 install wheel
+echo "*** Installing twine..."
+sudo -H pip3 install twine
+echo "*** Packaging and uploading..."
+cd _build/python
+python3 setup.py sdist bdist_wheel
+twine upload -u __token__ -p $pypi_token dist/*
```
Comment (JaviCerveraIngram) [inline stuff/pypi_deploy.sh]: Travis pypi_token env var is used here to upload the package.

File: Packager.hx [hunk 5]
```diff
@@ -179,50 +178,36 @@ class Packager {
 
 
     private static function createPhpPackage(classes: Array<String>): Void {
```
Comment (JaviCerveraIngram) [inline Packager.hx]: The code that has been removed is already autogenerated by Haxe when creating the PHP package.

File: Packager.hx (hunk 2 above)
Comment (marcserrat) [reply Packager.hx]: Correct

**Pull request merged**